internal/apptoken/query.go

```diff
@@ -166,10 +167,6 @@ left join iam_scope_project
      join iam_grant
        on app_token_permission_grant.canonical_grant = iam_grant.canonical_grant
       and iam_grant.resource = any(@resources)
-left join app_token_permission_org_individual_grant_scope individual_project_grants
-       on app_token_permission_org.private_id = individual_project_grants.permission_id
-left join iam_scope_project
-       on individual_project_grants.scope_id = iam_scope_project.scope_id
```

Comment:
Reason for this change: https://hashicorp.atlassian.net/browse/ICU-17918?focusedCommentId=907949

Reply:
Moved to https://github.com/hashicorp/boundary/pull/6291